docs/proposals/20191017-kubeadm-based-control-plane.md

```diff
@@ -336,7 +336,12 @@ spec:
   - Upgrading machines
 - Scale up operations are blocked based on Etcd and control plane health checks.
   - See [Health checks](#Health checks) below.
-- Scale down operations removes the oldest machine in the failure domain that has the most control-plane machines on it 
+- Scale down operations removes the oldest machine in the failure domain that has the most control-plane machines on it
+- Scale down operations can be also triggered by marking specific control plane machine(s) using delete annotation key.
```

Comment:
From what I understand, the annotation is a way to condition how to select which machine to delete when scaling down, but the annotation alone does not trigger scale down. 
If this is true, this sentence should be changed (or removed)